### ISSUE TITLE
Does this only localize the object?

### ISSUE THREAD

**Issue by Thilanka97** [AlexeyAB/yolo2_light#39]
@AlexeyAB Hey, I tried int8 version for tiny yolo2 but it only localizes the object but does not classify the class of the object. does this code only localize the objects and not classify ? 

Thanks in advance!

**Comment by AlexeyAB** [comment]
@Thilanka97 

It classufy and localize the objects, just don't print labels on the image.
See the console output to see labels of objects.

**Comment by Thilanka97** [comment]
@AlexeyAB  yes it shows the percentage but does not indicate the class. I do not know if I a doing anything wrong. But I did not change anything as such. 
Thanks in advance!

**Comment by Thilanka97** [comment]
@AlexeyAB  Hey, I am trying to run int8 version of tiny yolov2 as I mentioned and so far I used detector test for darknet as mentioned below,
./darknet detector test voc.data tiny-yolo-voc.cfg yolov2-tiny-voc.weights dog.jpg -thesh 0.25 -quantized

I am using trained weights for voc data set on yolo website. detector test works and gives percentages (but not classes as I mentioned in the previous comment).

But then I wanted to check the map(mean average precision) and used detector map command instead as below. 
./darknet detector map voc.data tiny-yolo-voc.cfg yolov2-tiny-voc.weights dog.jpg -thesh 0.25 -quantized

but this gives me an error saying "Couldn't open file: data/2007_test.txt". Then I checked in the official darknet yolo website and it says to run voc_label.py script to generate this file. But that is for training I guess. Anyways I tried that and it also gives me an error saying "No such file or directory: 'VOCdevkit/VOC2012/ImageSets/Main/train.txt'   "

how can I use map? please explain if possible. would be a great help.

Thanks in advance!

**Comment by Thilanka97** [comment]
@AlexeyAB hey I got this working.

**Comment by AlexeyAB** [comment]
@Thilanka97 Hi,

There is manual how to check mAP on Pascal VOC: https://github.com/AlexeyAB/darknet/tree/47c7af1cea5bbdedf1184963355e6418cb8b1b4f#how-to-calculate-map-on-pascalvoc-2007

**Comment by abhigoku10** [comment]
@Thilanka97 i am trying to run the quantized for yolov2 and tiny yolov2 can you please share teh cfg and weight files to mail id abhigoku10@gmail.com . I am not able to get any output from the model which i tried with
